build/cqs.tsv

```diff
@@ -4,9 +4,8 @@ com.beust:jcommander	1.48	compile
 com.carrotsearch:hppc	0.7.2	compile
 com.clearspring.analytics:stream	2.9.2	compile
 com.drewnoakes:metadata-extractor	2.8.1	compile
-com.esotericsoftware.kryo:kryo	2.21	compile
+com.esotericsoftware.kryo:kryo	2.24.0	compile
```

Comment:
i wonder if we're going to have any problems with kryo... was there a reason to bump the version?
https://rawgit.com/EsotericSoftware/kryo/master/compat_reports/kryo/2.22_to_2.23.0/compat_report.html
https://rawgit.com/EsotericSoftware/kryo/master/compat_reports/kryo/2.23.0_to_2.24.0/compat_report.html

Reply:
it looks like spark has bumped to twitter-chill-0.8, which includes kryo 3.0.3 - we've still got twitter-chill pegged to 0.5, should probably update that